client/payment-details/dispute-details/dispute-steps.tsx

```diff
@@ -28,6 +28,42 @@ interface Props {
 	daysRemaining: number;
 }
 
+const DueByDate: React.FC< {
```

Comment:
I moved this out to a standalone component, since it was quite complex logic and it identical for both cases.